src/SFA.DAS.ApprenticeCommitments.Web/Api/RegistrationsService.cs

```diff
@@ -11,5 +11,8 @@ public class RegistrationsService
         public RegistrationsService(IApiClient client) => _client = client;
 
         public Task<Registration> GetRegistration(Guid id) => _client.GetRegistration(id);
+
+        internal Task Validate(VerifyRegistrationCommand verification) =>
+            _client.Validate(verification.RegistrationId, verification);
     }
```

Comment:
I was thinking we could let the outer api do the validations and we could capture the errors and then map them to the appropriate model fields for display

Reply:
Yes, this call here is actually the call to the outer API - badly named!  It should be `VerifyRegistration`